marine/src/config/marine_config.rs

```diff
@@ -285,12 +304,25 @@ impl<'c> TryFrom<WithContext<'c, TomlWASIConfig>> for MarineWASIConfig {
             Ok((elem.0.into_bytes(), to.into_bytes()))
         };
 
+        let check_path = |path: PathBuf| -> Result<PathBuf, Self::Error> {
+            if path.is_absolute() {
+                return Err(MarineError::InvalidConfig(format!("Absolute paths are not supported in WASI section: {}", path.display())));
```

Comment:
I don't understand. Does that mean I cannot do
```
mapped_dirs = { "/tmp" = "/tmp" }
```
anymore?

Reply:
Yes. 
You should still be able to do this:
```mapped_dirs = { "/tmp" = "tmp" }```